docs/website/docs/building-from-source/optional-features.md

```diff
@@ -5,41 +5,44 @@ Most of these are controlled by various CMake options, sometimes requiring
 extra setup or preparation. Each section extends the basic build steps
 in the [getting started](../getting-started/) page.
 
-## Building Python Bindings
+## Building Python bindings
 
 This section describes how to build and interactively use built-from-source
 Python bindings for the following packages:
 
-| Python Import             | Description                                                                 |
-|------------------------------|-----------------------------------------------------------------------------|
-| `import iree.compiler`     | IREE's generic compiler tools and helpers                                   |
-| `import iree.runtime`      | IREE's runtime, including CPU and GPU backends                              |
+| Python Import          | Description                                    |
+|------------------------|------------------------------------------------|
+| `import iree.compiler` | IREE's generic compiler tools and helpers      |
+| `import iree.runtime`  | IREE's runtime, including CPU and GPU backends |
```

Comment:
(I don't have an opinion on the formatting here, but my editor does - can revert if that's preferred)